src/collection.js

```diff
@@ -408,7 +408,10 @@ export default class Collection {
       return Promise.reject(new Error(`Invalid Id: ${record.id}`));
     }
     return this.get(record.id)
-      .then(_ => {
+      .then(res => {
+        if (res.data.last_modified && !record.last_modified) {
+          record.last_modified = res.data.last_modified;
+        }
```

Comment:
I wonder if this is still valid with changes of #310.
Also shouldn't we avoid `record` mutation and create a copy instead?